### PR TITLE
[WIP] Declarative FormSet

### PR DESCRIPTION
**Pull request by auvipy**

### CODE REVIEW

File: django/contrib/contenttypes/management.py [hunk 1]
```diff
@@ -13,14 +13,14 @@ def update_contenttypes(app_config, verbosity=2, interactive=True, using=DEFAULT
         return
 
     try:
-        ContentType = apps.get_model('contenttypes', 'ContentType')
+        contenttype = apps.get_model('contenttypes', 'contenttype')
```
Comment (hobarrera) [inline django/contrib/contenttypes/management.py]: This actually returns a class, so the CamelCase is correct.

Comment (auvipy) [inline django/contrib/contenttypes/management.py]: thank you. I will apply suggested changes

**Comment by hobarrera** [comment]
@timgraham: Why has this been closed? It seemed like a useful addition.

**Comment by timgraham** [comment]
It's not ready for review.

**Comment by hobarrera** [comment]
Oh, thanks (I was unaware that WIP branches were not acceptable). :D

**Comment by auvipy** [comment]
I will push again when it will be ready for review soon. @hobarrera

**Pull request closed without merging**